Python/asm_trampoline.S

```diff
@@ -34,3 +37,21 @@ _Py_trampoline_func_start:
     .globl	_Py_trampoline_func_end
 _Py_trampoline_func_end:
     .section        .note.GNU-stack,"",@progbits
+#if defined(__x86_64__) && defined(__CET__) && (__CET__ & 1)
+    .section    .note.gnu.property,"a"
```

Comment:
What is the purpose of this section? Would you mind to add a comment?

Reply:
Added a comment.